### ISSUE TITLE
Album already exists but doesn't show in getAlbums

### ISSUE THREAD

**Issue by jake-newsom** [capacitor-community/media#59]
**Describe the bug**
I check to see if the album exists and if not create it, then save a video to it. The album now exists in my phone's gallery and the video file is there. However, when I run this code again, the album does not show in getAlbums(), so then my code attempts to create the album, but it says albumAlready exists. 

I've seen the other reports about this error, which say the issue was resolved in 5.2.1 ... I am using 5.2.1 but this issue is slightly different as it's not returning the album in getAlbums() despite the album existing with a file in it (not an empty album) 

**To Reproduce**
- Here is the code I'm running my Ionic app: 

```javascript
    async getAlbum(name: string): Promise<MediaAlbum> {
      console.log("Get album: ", name);
      let albums = null;
      let album = null;

      albums = await Media.getAlbums();
      console.log("Albums: ", albums);
      album = albums.albums.find((a) => a.name === name);

      if (!album) {
        const createAlbumResponse = await Media.createAlbum({ name: name });
        console.log("Create album: ", createAlbumResponse);
        albums = await Media.getAlbums();
        console.log("albums: ", albums);
        album = albums.albums.find((a) => a.name === name);
      }

      console.log("Found this album: ", album);

      return album;
    },

    async downloadRealTalkAndroid(track: boolean) {
    
      const album = await this.getAlbum(this.albumName);
      if (!album) {
        console.log("Unable to get album");
        return;
      }
      console.log("In download method we got this album...:", album);

      try {
        const options: MediaSaveOptions = {
          path: this.decodedFilePath,
          albumIdentifier: album.identifier,
        };
        console.log("Media save options: ", options);

        Media.saveVideo(options);
        this.hasSavedVideo = true;
        const toast = await toastController.create({
          message: this.t("realtalk.downloading"),
          duration: 2000,
        });
        toast.present();
      } catch (error) {
        const toast = await toastController.create({
          message: this.t("realtalk.downloadingError"),
          duration: 2000,
        });
        toast.present();
      }
    },
```
    
This code runs once, and executes fine. The album is created and the file is saved there. Then the very next time I run this code it errors saying the album already exists and can't be created (despite not showing in getAlbums)

**Expected behavior**
I would expect the newly created folder with a file in it to show up in getAlbums(). I do see these albums: 
{name: 'Download', identifier: '/storage/emulated/0/Download'}
{name: 'Camera', identifier: '/storage/emulated/0/DCIM/Camera'}
{name: 'WhatsApp Images', identifier: '/storage/emulated/0/Android/media/com.whatsapp/WhatsApp/Media/WhatsApp Images'}
{name: 'Messages', identifier: '/storage/emulated/0/Pictures/Messages'}
{name: 'Facebook', identifier: '/storage/emulated/0/DCIM/Facebook'}
{name: 'Snapchat', identifier: '/storage/emulated/0/DCIM/Snapchat'}
{name: 'Pictures', identifier: '/storage/emulated/0/Pictures'}
{name: 'DCIM', identifier: '/storage/emulated/0/DCIM'}
{name: 'PokemonGO', identifier: '/storage/emulated/0/Pictures/PokemonGO'}
{name: 'Screenshots', identifier: '/storage/emulated/0/Pictures/Screenshots'}
{name: 'Restored', identifier: '/storage/emulated/0/DCIM/Restored'}
{name: 'Reddit', identifier: '/storage/emulated/0/Pictures/Reddit'}
{name: 'PhotosEditor', identifier: '/storage/emulated/0/DCIM/PhotosEditor'}
{name: 'Lensa', identifier: '/storage/emulated/0/Pictures/Lensa'}
{name: 'Canva', identifier: '/storage/emulated/0/Pictures/Canva'}
{name: 'Documents', identifier: '/storage/emulated/0/Documents'}
{name: 'Giphy', identifier: '/storage/emulated/0/Pictures/Giphy'}
{name: 'Instagram', identifier: '/storage/emulated/0/Pictures/Instagram'}

However the recently created one is not showing, this was the response from createAlbum:
{name: 'X', identifier: '/storage/emulated/0/Android/media/com.x.x/X'}

**Smartphone (please complete the following information):**
 - Device: Google Pixel 7 Pro
 - OS: Android 13
 - Version: 5.2.1

**Comment by nkalupahana** [comment]
@jake-newsom Are you able to reproduce this issue with the example app? (Click create demo album -- then, does it show up with get albums?) The example app is in the `example/` directory.

**Comment by jake-newsom** [comment]
@nkalupahana Hey thanks for your time!

I ran the demo and went through these steps: 

1. Get Albums - this worked and showed all my albums (except the one from my app, with the video file in it) 
2. Create Demo Album - this worked and added the Media Demo Album (verified via Get Albums button)
3. Create Random Demo Album - this worked and added a random album (verified via Get Albums button) 
4. Save Previously Taken Video - my apps video file shows in the list here, I chose that file. It looks like it saved because now when I press "Save Previously Taken Video" it shows up twice
5. Get Albums - NOW the demo one does not show at all
6. Create Demo Album - Does nothing now
7. Save Previously Taken Video - says "Demo album does not exist; create it first using "Create Demo Album" button above"
